Sources/TuistSupport/System/System.swift

```diff
@@ -243,8 +243,7 @@ public final class System: Systeming {
     public func run(_ arguments: [String]) throws {
         let process = Process(arguments: arguments,
                               environment: env,
-                              outputRedirection: .stream(stdout: { _ in },
-                                                         stderr: { _ in }),
+                              outputRedirection: .collect,
```

Comment:
Is this necessary?

Reply:
Otherwise, there is not output in `throwIfErrored()` function, so I'd say this change is for the better or is there a reason why not to do this?